cmd/admin-router.go

```diff
@@ -166,7 +166,15 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 
 			// Set Group Status
 			adminRouter.Methods(http.MethodPut).Path(adminVersion+"/set-group-status").HandlerFunc(httpTraceHdrs(adminAPI.SetGroupStatus)).Queries("group", "{group:.*}").Queries("status", "{status:.*}")
-
+			// GetBucketQuotaConfig
```

Comment:
This should be disabled in gateway mode @poornas

Reply:
done,thanks